samples/net/sockets/http_get/src/http_get.c

```diff
@@ -66,12 +81,16 @@ int main(void)
 	CHECK(sock);
 	printf("sock = %d\n", sock);
 	CHECK(connect(sock, res->ai_addr, res->ai_addrlen));
-	send(sock, REQUEST, SSTRLEN(REQUEST), 0);
 
-	printf("Response:\n");
+	zstream_sock_init(&stream_sock, sock);
+	stream = (zstream)&stream_sock;
+
+	stream->api->write(stream, REQUEST, SSTRLEN(REQUEST));
```

Comment:
So, that's how it looks. https://github.com/zephyrproject-rtos/zephyr/issues/5900 mentioned this matter, now we can consider exact variants of look&feel:

~~~
stream->vt->write(stream, ...) - from "vtable"
stream->meth->write(stream, ...) - from "method"
~~~

These aren't much better/different from `api` IMHO. Note that besides these (repeating), and extra var also needs to be introduced: `stream = (zstream)&stream_sock`.

~~~
CALL_METH(stream, write, ...)
~~~

This still looks uglier IMHO. Also, realistically that would be ZSTREAM_CALL_METH().

Reply:
Meth. Not even once.

Reply:
Using the stream struct like it is super ugly actually. I'd rather see inline functions to hide this.

Reply:
> I'd rather see inline functions to hide this.

Indeed, the winner is: zstream_write(), zstream_read(), etc. I'll add them.